amazonlinux1.dockerfile

```diff
@@ -28,20 +28,20 @@ RUN curl -fL https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0
   && mv /tmp/clang+llvm-10.0.0-x86_64-linux-sles11.3 /tmp/clang
 ENV PATH=/tmp/clang-llvm/bin:$PATH
 
-ENV RUST_VERSION=1.44.0
+ENV RUST_VERSION=1.45.0
 RUN curl https://sh.rustup.rs -sSf \
   | sh -s -- --default-toolchain ${RUST_VERSION} -y
 ENV PATH=/root/.cargo/bin:$PATH
 
-ENV DENO_VERSION=1.2.1
+ENV DENO_VERSION=1.2.2
 
 RUN curl -fsSL https://github.com/denoland/deno/releases/download/v${DENO_VERSION}/deno_src.tar.gz \
   --output deno.tar.gz \
   && tar -zxf deno.tar.gz \
   && rm deno.tar.gz
 
 RUN yum install -y gcc-c++ libatomic
-RUN echo "INPUT ( /usr/lib64/libatomic.so.1.2.1 )" \
+RUN echo "INPUT ( /usr/lib64/libatomic.so.1.2.2 )" \
```

Comment:
Not sure I want to know how you worked this out 😳

Reply:
Oh, this was a sed error I guess. The fact it wasn't necessary is interesting... Presumably one of the many lambda tests would have failed if it were.

Hmm!